### PR TITLE
Accelerating convolution by changing the way of memory access.

### PR DESCRIPTION
**Pull request by MambaWong**

### CODE REVIEW

File: modules/dpm/src/dpm_convolution.cpp [hunk 1]
```diff
@@ -48,12 +48,28 @@ namespace dpm
 double ConvolutionEngine::convolve(const Mat &feat, const Mat &filter,
         int dimHOG, int x, int y)
 {
+    // // Inherited from Matlab
```
Comment (sovrasov) [inline modules/dpm/src/dpm_convolution.cpp]: I think the commented code can be deleted.

Comment (MambaWong) [inline modules/dpm/src/dpm_convolution.cpp]: Yes, I will delete the commented code.

**Comment by sovrasov** [comment]
@MambaWong please squash your commits into one before merge.

**Comment by MambaWong** [comment]
@sovrasov Done!

**Comment by sovrasov** [comment]
:+1:

**Pull request merged**